### PR TITLE
Updated to explain geo placement logic.

### PR DESCRIPTION
**Pull request by keithchambers**

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: yammer/manage-security-and-compliance/data-residency.md [hunk 1]
```diff
@@ -19,11 +19,9 @@ description: "Data residency for Yammer Enterprise."
 ---
 
 # Data residency
-Yammer offers local data residency to help meet data residency requirements. We commit to store message bodies and files attached to messages at rest within a specific geographical area (Geo). The country you enroll in Office 365 from will determine which Geo your Yammer Enterprise network is associated with. For a new Yammer Enterprise network, Microsoft will store Yammer message bodies and files in legacy Yammer Groups at rest in the Yammer Enterprise network Geo. 
+Yammer offers local data residency to help meet data residency requirements. We commit to store message bodies and files attached to messages at rest within a specific geographical area (Geo). Yammer files are saved either in Yammer cloud storage, or for Office 365 connected groups, some Yammer files are stored in SharePoint. Yammer files saved in SharePoint will be stored in SharePoint Online per your SharePoint Online data residency policy. Mobile push notifications require sending data to a third party notification service (Apple or Google), which might be outside your Geo.
```
Comment (nenonix) [inline yammer/manage-security-and-compliance/data-residency.md]:
```suggestion
Yammer offers local data residency to help meet data residency requirements. We commit to store message bodies, and files attached to messages at rest, within a specific geographical area (Geo). Yammer files are saved in either Yammer cloud storage, or for Office 365 connected groups, some files are stored in SharePoint. Yammer files saved in SharePoint will be stored in SharePoint Online per your SharePoint Online data residency policy. Mobile push notifications, which require sending data to a third party notification service (Apple or Google), might be outside your Geo.
```

File: yammer/manage-security-and-compliance/data-residency.md [hunk 2]
```diff
@@ -19,11 +19,9 @@ description: "Data residency for Yammer Enterprise."
...
 
-Yammer files are saved either in Yammer cloud storage, or for Office 365 connected groups, some Yammer files are stored in SharePoint. Yammer files saved in SharePoint will be stored in SharePoint Online per your SharePoint Online data residency policy.
-
-Mobile push notifications require sending data to a third party notification service (Apple or Google), which might be outside your Geo.
+Your Yammer Enterprise network is automatically created when you create your Office 365 tenant. For Office 365 Education subscribers,  your network is associated with the US Geo. For all other Office 365 subscribers, the country you enroll from determines the Geo your network is associated with. When you enroll from Europe or Africa your network is associated with the EU Geo and when you enroll from Australia, Asia, North America, or South America your network is associated with the US Geo.
```
Comment (nenonix) [inline yammer/manage-security-and-compliance/data-residency.md]:
```suggestion
Your Yammer Enterprise network is automatically created when you create your Office 365 tenant. For Office 365 Education subscribers,  your network is associated with the US Geo. For all other Office 365 subscribers, the country you enroll from determines the Geo your network is associated with. When you enroll from Europe or Africa,  your network is associated with the EU Geo. When you enroll from Australia, Asia, North America, or South America. your network is associated with the US Geo. 
```

**Comment by mypil** [comment]
@keithchambers -  we need your kind assistance to merge the new commit for the copy/edit review made by JohanFreelancer9 / nenonix or make the necessary changes on the PR following the suggestions made. 

Thank you.

**Comment by cmcatee-MSFT** [comment]
@IrenePasternack can you please review these changes & approve or reject the PR? Thanks.

**Pull request merged**